02-technical-guides/hierarchies/01-hierarchy-fundamentals.ipynb

```diff
@@ -0,0 +1,1592 @@
+{
```

Comment:
<blockquote>however, for the purposes of this walk through, we've elected to use <code>session.widget</code> so that we can show our analysis directly within this Jupyter notebook.</blockquote><p>No need to change this here, just food for thought:</p><p>Yes, we use the extension for creating a story in a Jupyter notebook, but the extension also provides value to other users, allowing them to do iterative analysis, et cetera. We should be careful when we talk about it that we also highlight how this extension exists as a tool to help them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Good point, I changed this to the following to accommodate what you mentioned:</p><p> </p><p><strong>💡&nbsp;Note: You may also use&nbsp;</strong><code><strong>session.link</strong></code><strong>&nbsp;to perform the same investigation within the Atoti UI; however, for the purposes of this walk through, we've elected to use&nbsp;</strong><code><strong>session.widget</strong></code><strong>&nbsp;so that we can show our analysis directly within this Jupyter notebook and provide storytelling as we create several visualizations in Atoti.</strong></p>

Reply:
Let's stick to "we" instead of "you" in our writing. The usage of "you" should be rare, and only if necessary.

Reply:
Fixed!